src/migrations/201602231400_initial.js

```diff
@@ -25,6 +25,8 @@ function createFulfillmentsTable (knex) {
 function createEntriesTable (knex) {
   return knex.schema.createTableIfNotExists('entries', (table) => {
     table.increments()
+    // A FK references to transfers table will cause an error here
+    // there are cases in which an entry is inserted before the corresponding transfer is inserted
```

Comment:
We need to revisit FK constraints. Frequently, oracle gives ORA-02449 when dropping tables. We need to drop with some kind of cascade in the `migrate:rollback`.